app/src/main/java/cash/z/ecc/android/ui/send/SendFragment.kt

```diff
@@ -172,8 +186,8 @@ class SendFragment : BaseFragment<FragmentSendBinding>(),
     }
 
     /**
-    * To hide input Memo and reply-to option for T type address and show a info message about memo option availability
-    * */
+     * To hide input Memo and reply-to option for T type address and show a info message about memo option availability
+     * */
```

Comment:
minor: there seems to be an extra asterisk here that we can get rid of (This PR didn't add it but since the line is changing we might want to correct the formatting).
```suggestion
     */
```

Reply:
Fixed.